components/accelerator/nvidia/bad-envs/component.go

```diff
@@ -52,6 +52,22 @@ func New(gpudInstance *components.GPUdInstance) (components.Component, error) {
 
 func (c *component) Name() string { return Name }
 
+func (c *component) Labels() []string {
+	return []string{
+		"accelerator",
+		"gpu",
+		"nvidia",
+		Name,
```

Comment:
It seems that these are tags not labels. Labels are composed of keys and values..

Reply:
Yes, actually, tags make more sense. Reverted